### ISSUE TITLE
llama-7B-qlora result

### ISSUE THREAD

**Issue by AdjugateMatrix** [eosphoros-ai/DB-GPT-Hub#54]
```

                     easy                 medium               hard                 extra                all                 
count                248                  446                  174                  166                  1034                
compare etype exec
=====================   EXECUTION ACCURACY     =====================
execution            0.786                0.675                0.563                0.313                0.625
````
The above  is what I got based on llama-7b and adapter/checkpoint-10000(qlora), it seems the result(0.625) is better than llama2_13b_hf_lora(0.622 in your review)?Dose that mean the base model is not very important..

**Comment by wangzaistone** [comment]
I don't thinks the base model is not very important . And  thanks  a lot for your share, it's a great work for us to refer. the default  llama2_13b_hf_lora(0.622) in our exp is just trained checkpoint-500(lora) . it's  seems not trained enough。
Can you share your param  details  like learning rate and so on  about the  result(0.625) ?

**Comment by wangzaistone** [comment]
> ```
> 
>                      easy                 medium               hard                 extra                all                 
> count                248                  446                  174                  166                  1034                
> compare etype exec
> =====================   EXECUTION ACCURACY     =====================
> execution            0.786                0.675                0.563                0.313                0.625
> ```
> 
> The above is what I got based on llama-7b and adapter/checkpoint-10000(qlora), it seems the result(0.625) is better than llama2_13b_hf_lora(0.622 in your review)?Dose that mean the base model is not very important..

I don't thinks the base model is not very important . And thanks a lot for your share, it's a great work for us to refer. the default llama2_13b_hf_lora(0.622) in our exp is just trained checkpoint-500(lora) . it's seems not trained enough。
Can you share your param details like learning rate and so on about the result(0.625) ? It would be greate helpful for all  guys who want to do the experiments!

**Comment by JasonLLLLLLLLLLL** [comment]
> Collaborator

Actually I just use those default params, since I am not familar the project and LLM ... I thought those default params are good engough

**Comment by JasonLLLLLLLLLLL** [comment]
> > ```
> > 
> >                      easy                 medium               hard                 extra                all                 
> > count                248                  446                  174                  166                  1034                
> > compare etype exec
> > =====================   EXECUTION ACCURACY     =====================
> > execution            0.786                0.675                0.563                0.313                0.625
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > The above is what I got based on llama-7b and adapter/checkpoint-10000(qlora), it seems the result(0.625) is better than llama2_13b_hf_lora(0.622 in your review)?Dose that mean the base model is not very important.
> 
> I don't thinks the base model is not very important . And thanks a lot for your share, it's a great work for us to refer. the default llama2_13b_hf_lora(0.622) in our exp is just trained checkpoint-500(lora) . it's seems not trained enough。 Can you share your param details like learning rate and so on about the result(0.625) ? It would be greate helpful for all guys who want to do the experiments!

Actually I just use those default params, since I am not familar the project and LLM, I thought those default params are good engough, so I just use those command provided in the README and I don't change them. like max_steps = 10000....orz, but it seems there is no need to use that many steps, since you got a good result by 500 steps.

I wonder are those default tested or checked on your computer before? 

Thanks, and this is really a good project

**Comment by jcdiv47** [comment]
I was wondering if any of you ran into the follow issues during prediction, which I did with random sample test data:

- the model tried to use a table that did not exist, usually out of its own speculation
- the model tried to use a column that did not exist in the table
- the model selected the correct column, but from the wrong table
- etc. 

I was just curious as to what might be most probable cause here. Is it because I overfitted model or maybe underfitted? Or do I need to continue training with another dataset?

**Comment by JasonLLLLLLLLLLL** [comment]
> 
 
I write my own data to sql_fine_data.json, and then it can read my own data

**Comment by JasonLLLLLLLLLLL** [comment]
> > 
> 
> I write my own data to sql_fine_data.json, and then it can read my own data

I write my own data to sql_fine_data.json, and then it can read my own data

**Comment by JasonLLLLLLLLLLL** [comment]
> I was wondering if any of you ran into the follow issues during prediction, which I did with random sample test data:
> 
> * the model tried to use a table that did not exist, usually out of its own speculation
> * the model tried to use a column that did not exist in the table
> * the model selected the correct column, but from the wrong table
> * etc.
> 
> I was just curious as to what might be most probable cause here. Is it because I overfitted model or maybe underfitted? Or do I need to continue training with another dataset?

I write my own data to sql_fine_data.json, and then it can read my own data
